tool/requote.js

```diff
@@ -273,16 +271,16 @@ function requoteSentence(id, sentence, program, mode) {
     sentence = sentence.split(' ');
     program = program.split(' ');
 
-    let [spansBySentencePos, spansByProgramPos_sorted, sortIndices] = findSpanPositions(id, sentence, program)
+    let [spansBySentencePos, spansByProgramPos_sorted, sortIndices] = findSpanPositions(id, sentence, program);
```

Comment:
By the way, it's very jarring to mix snake_case and camelCase _in the same variable_. Please pick one or the other - preferrably camelCase unless it's a key JSON or a Thingpedia parameter.

Reply:
Yeah, the pythonic way of thinking sn(e)aked in here...